nativewind-env.d.ts

```diff
@@ -1 +1,16 @@
 /// <reference types="nativewind/types" />
+
+declare module "react-native-country-flag" {
```

Comment:
this should be in its own .d.ts file

Reply:
Fixed by doing this is a better way in https://github.com/getAlby/go/pull/331/commits/1808c35794f9ca13a6b9788247ca00ff7aaf971d